base/resources/home/.config/Code/User/argv.json

```diff
@@ -0,0 +1,3 @@
+{
+    "locale":"en"
+}
```

Comment:
End text files with a newline. _You can set most editors to do this automatically on save._